src/actix/api/service_api.rs

```diff
@@ -117,6 +119,29 @@ async fn kubernetes_healthz() -> impl Responder {
         .body("healthz check passed")
 }
 
+#[get("/logger")]
+async fn get_logger_config(handle: web::Data<tracing::LoggerHandle>) -> impl Responder {
+    let timing = Instant::now();
+    let result = handle.get_config().await;
+    process_response(Ok(result), timing)
+}
+
+#[post("/logger")]
+async fn update_logger_config(
+    handle: web::Data<tracing::LoggerHandle>,
+    config: web::Json<tracing::LoggerConfigDiff>,
+) -> impl Responder {
+    let timing = Instant::now();
+
+    let result = handle
+        .update_config(config.into_inner())
+        .await
+        .map(|_| true)
+        .map_err(|err| StorageError::service_error(err.to_string()));
+
+    process_response(result, timing)
+}
```

Comment:
I think that we should add the OpenAPI definitions in this PR as well.

Reply:
It's in the TODO list in PR description. :D